docs/using/config/configuration.rst

```diff
@@ -311,7 +311,7 @@ Here's an example of a simple but complete configuration defining a deployment o
 
   "version": 1
   "nodes":
-    "node017.example.com":
+    "172.16.255.251":
```

Comment:
:question: Maybe use an IPv4 address reserved for documentation:
 * https://tools.ietf.org/html/rfc5737#section-3